skopt/searchcv.py

```diff
@@ -44,7 +44,7 @@ class BayesSearchCV(BaseSearchCV):
         or ``scoring`` must be passed.
 
     search_spaces : dict, list of dict or list of tuple containing
-        (dict, int), or None.
+        (dict, int).
         One of 4 following cases:
```

Comment:
-> "One of these cases:" (don't have four anymore and this way we won't have to update the text again when a new one is added ;))